solidus_active_shipping.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'solidus_support'
   s.add_dependency 'solidus_backend', solidus_version
   s.add_dependency 'solidus_api', solidus_version
-  s.add_dependency 'active_shipping', '~> 1.13'
+  s.add_dependency 'active_shipping', '~> 2.1.1'
```

Comment:
This potentially breaks peoples code. So what about

```rb
s.add_dependency 'active_shipping', ['>= 1.13', '< 3']
```

This way we still allow 1.x and also allow 2.x